test/attr/attr_availability_osx.swift

```diff
@@ -13,22 +13,22 @@ enum Optional<T> {
 func doSomething() { }
 // expected-note @-1{{'doSomething()' was obsoleted in OS X 10.9}}
 
-doSomething() // expected-error{{'doSomething()' is unavailable: you don't want to do that anyway}}
+doSomething() // expected-error{{'doSomething()' is unavailable on OS X: you don't want to do that anyway}}
```

Comment:
Can we please make sure this says macOS before this gets merged?

Reply:
I think this uses whatever platform text was used in the `availability` attribute. In this case we still support `OSX` as an available platform, and the test case uses `OSX`. I imagine if you used `macOS` instead it will diagnose accordingly.

Reply:
No, this does actually use a name chosen by the compiler: `Attr->prettyPlatformString()`. I'll take a look at making that change.